src/library.js

```diff
@@ -4548,9 +4567,9 @@ LibraryManager.library = {
 
   emscripten_with_builtin_malloc__deps: ['emscripten_builtin_malloc', 'emscripten_builtin_free', 'emscripten_builtin_memalign'],
   emscripten_with_builtin_malloc: function (func) {
-    var prev_malloc = _malloc;
-    var prev_memalign = _memalign;
-    var prev_free = _free;
+    var prev_malloc = typeof _malloc !== 'undefined' ? _malloc : undefined;
+    var prev_memalign = typeof _memalign !== 'undefined' ? _memalign : undefined;
+    var prev_free = typeof _free !== 'undefined' ? _free : undefined;
```

Comment:
why are these changes needed?

Reply:
In MINIMAL_RUNTIME code malloc/memalign/free might not be getting pulled in to the build, if not used in the program.